source/_data/sidebar.yml

```diff
@@ -1,11 +1,14 @@
 docs:
   getting_started:
     overview: index.html
+    installation: installation.html
     quick_start: quick_start.html
 
   general_usage:
     creating_project: create_project.html
     structure: structure.html
+    running_apps: running_apps.html
```

Comment:
This is a new guide that focuses on things like options for `embark run` as well as environments and the simulator.